app/assets/javascripts/passengers.js

```diff
@@ -0,0 +1,15 @@
+$( document ).on("turbolinks:load", function() {
+  $('#passenger_spire').focusout(function(){
+    $.ajax({
+      type: 'GET',
+      url: '/passengers/check_existing',
+      data: { spire_id: $(this).val() },
+      success: function(response) {
+        if(response == undefined){ return }
```

Comment:
This seems odd. Why would the response be `undefined`?

Reply:
It's `undefined` when the controller responds with nothing. Open to suggestions to make it less weird.

Reply:
Eh. Maybe my objection is just about naming?

The "response" is there, it's the response _body_ (or whatever) that's blank.